### ISSUE TITLE
Google Cloud deploy script fails due to non existing os image

### ISSUE THREAD

**Issue by zenonasz** [btcpayserver/btcpayserver-googlecloud#10]
google cloud has end the support for  ubuntu-1604-lts (https://cloud.google.com/compute/docs/eol/ubuntu1604) as a result the script fails to create the VM-instance.

Any other os image supported and tested?

**Comment by Tragic** [comment]
+1 same issue

**Comment by MatapaCoin** [comment]
Hi guys
You edit the file main.btcpay.yaml and in linux Type: you change ubuntu-1604-lts to ubuntu-1804-lts and voila!
Best regards!

**Comment by zenonasz** [comment]
Indeed, that's what i did but this should be fixed in the repo, that's why I reported it.

**Comment by MatapaCoin** [comment]
> Indeed, that's what i did but this should be fixed in the repo, that's why I reported it.

It worked OK for you with version 18.04?

**Comment by zenonasz** [comment]
> > Indeed, that's what i did but this should be fixed in the repo, that's why I reported it.
> 
> It worked OK for you with version 18.04?

still setting it up but yes, its ok
